database/index.js

```diff
@@ -0,0 +1,17 @@
+const { Client } = require('pg');
+
+const client = new Client({
+  user: 'root',
```

Comment:
is there any way you can avoid posting your credentials to version control? This could be a security issue